Application/Client/src/app/mapData/mapData.ts

```diff
@@ -26,10 +26,10 @@ export class mapData{
         "ContactNo": "",
         "ContactName": "",
         "Configuration": "",
-        "Details": "",
+        "Details": (form.get('shortDesc')!=null)?form.get('shortDesc').value:'',
```

Comment:
expected 
form.get('shortDesc') ? form.get('shortDesc').value : ''

Also we can send only the filed data that we have and backend can entry default value null or empty